server/src/main/java/io/spine/server/command/model/CommandAssigneeMethod.java

```diff
@@ -41,31 +41,32 @@
 import static java.lang.String.format;
 
 /**
- * The wrapper for a command handler method.
+ * The wrapper for a method assigned to handle commands.
  */
-public final class CommandHandlerMethod
-        extends CommandAcceptingMethod<CommandHandler, EventClass>
-        implements EventProducingMethod<CommandHandler, CommandClass, CommandEnvelope> {
+public final class CommandAssigneeMethod
+        extends CommandAcceptingMethod<CommandAssignee, EventClass>
+        implements EventProducingMethod<CommandAssignee, CommandClass, CommandEnvelope> {
 
     /**
      * Creates a new instance to wrap {@code method} on {@code target}.
      *
      * @param method
-     *         command handler method
+     *         method assigned to handle commands
      */
-    CommandHandlerMethod(Method method, ParameterSpec<CommandEnvelope> params) {
+    CommandAssigneeMethod(Method method, ParameterSpec<CommandEnvelope> params) {
         super(method, params);
     }
 
     @Override
     public Success toSuccessfulOutcome(@Nullable Object rawResult,
-                                       CommandHandler target,
+                                       CommandAssignee target,
                                        CommandEnvelope handledSignal) {
         Success outcome =
                 EventProducingMethod.super.toSuccessfulOutcome(rawResult, target, handledSignal);
         if (outcome.getProducedEvents().getEventCount() == 0) {
             var errorMessage = format(
-                    "Command handler %s did not produce any events when processing command %s",
```

Comment:
This old version was good. Not sure why you changed it.

We could improve it by starting like that:
```
Command-handling method %s did not ...
```